.github/workflows/pipeline.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-        node-version: [12.x, 14.x]
+        node-version: [12.x, 14.x, 16.x]
```

Comment:
@kranthie-sap @donal-tobin-sap - Do we want to limit the windows build to only 1 node version?

Reply:
@devinea: Let's go with all 3 versions on all platforms for now. If the pipeline takes way too long to run, we can look at other options like running node v16 (non LTS version) on a scheduled run.